### PR TITLE
Formatting Stefan–Boltzmann constant

### LINKED ISSUE #13444: encoding missing from physics/units/definitions?
I get the following in 2.7 but not in Python 3.6:
```
Python 2.7.7 (default, Jun  1 2014, 14:17:13) [MSC v.1500 32 bit (Intel)] on win
32
Type "help", "copyright", "credits" or "license" for more information.
Traceback (most recent call last):
  File "\s\pythonstartup.py", line 18, in <module>
    from sympy.physics import units
  File "sympy\physics\__init__.py", line 5, in <module>
    from . import units
  File "sympy\physics\units\__init__.py", line 85, in <module>
    from .definitions import (
  File "sympy\physics\units\definitions.py", line 130
SyntaxError: Non-ASCII character '\xe2' in file sympy\physics\units\definitions.
py on line 130, but no encoding declared; see http://www.python.org/peps/pep-026
3.html for details
>>>
```

### PR DESCRIPTION
**Pull request by Shikhar1998**
@Upabjojr ping.

fixes #13444

### CODE REVIEW

**Comment by smichr** [comment]
Does this address #13444?

**Comment by Shikhar1998** [comment]
No, but a small formatting mistake in #13440

**Comment by Shikhar1998** [comment]
@jksuom, @smichr, I hope this will solve the problem

**Comment by gxyd** [comment]
I would prefer having "Stefan-Boltzmann constant" instead of "Stefan_Boltzmann constant". As can be see on wikipedia article https://en.wikipedia.org/wiki/Stefan%E2%80%93Boltzmann_constant .

**Comment by smichr** [comment]
> Stefan-Boltzmann constant

agree. And I think this solves #13444 since that error occurs on line 130...which is the one you have edited.

**Comment by smichr** [comment]
(I stopped the build since an edit will soon follow by you.)

**Comment by jksuom** [comment]
Wikipedia seems to use one of the utf-8 versions of dash, but we could use an ordinary ascii minus sign here. (They look the same on a utf-8 screen.)

File: sympy/physics/units/definitions.py [hunk 1]
```diff
@@ -127,7 +127,7 @@
 avogadro = avogadro_constant = Quantity("avogadro_constant", amount_of_substance**-1, avogadro_number / mol)
 # Boltzmann constant
 boltzmann = boltzmann_constant = Quantity("boltzmann_constant", energy/temperature, 1.38064852e-23*joule/kelvin)
-#Stefan–Boltzmann constant
+# Stefan-Boltzmann constant
```
Comment (smichr) [inline sympy/physics/units/definitions.py]: there was the problem: a special dash instead of an ASCII minus.

**Pull request merged**